decidim-core/app/cells/decidim/card_metadata_cell.rb

```diff
@@ -75,7 +75,7 @@ def coauthors_item
 
       {
         cell: "decidim/coauthorships",
-        args: [resource, { stack: true }]
+        args: [resource, { stack: true, context_actions: [] }]
```

Comment:
I think this is not necessary. At least removing it and going to the proposals show page, I still see the author with the date

```suggestion
        args: [resource, { stack: true }]
```

Reply:
Actually, this is required to be like this. The cell is automatically adding created / published at and emendation, and we push this empty array, so that we do not have duplicated data on the proposal cards in index view ( you may need to reset cache to properly see the error )

Reply:
Ah Ok, I missed that bug in the cards. Nice catch! Let's merge this as is then